scripts/data.example.js

```diff
@@ -915,6 +915,45 @@ module.exports = {
         }
       ]
     },
+
+    onStripePaymentSuccess: {
+      name: 'On Stripe payment success',
+      description: 'Handles series of actions after a successful payment',
```

Comment:
It would be more useful to describe the origin of the event, and explicitely state that the workflow passes the paymont on to Stelace by creating `transaction` record.

Reply:
You're right.